Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.13.1-alpine3.21
+
+LABEL maintainer="gnonasis@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/

```

Comment:
The `WORKDIR` should be set to an absolute path, such as `/app`, to ensure the directory is created and used correctly. This can prevent potential issues with relative paths.